<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Function</class>
 <widget class="QDialog" name="Function">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1261</width>
     <height>891</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Top POIs</string>
    </attribute>
    <widget class="QChartView" name="chartTop" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>1221</width>
       <height>641</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>31</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Select User:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="top_user_Box">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>77</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="top_start_edit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>230</width>
       <height>35</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2009</year>
       <month>2</month>
       <day>4</day>
      </datetime>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Start Time</string>
     </property>
    </widget>
    <widget class="QSlider" name="top_start_time">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>230</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="top_end_time">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>120</y>
       <width>230</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>626</number>
     </property>
     <property name="sliderPosition">
      <number>626</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>41</y>
       <width>177</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>End Time</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="top_end_edit">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>70</y>
       <width>230</width>
       <height>35</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2010</year>
       <month>10</month>
       <day>23</day>
      </datetime>
     </property>
    </widget>
    <widget class="QPushButton" name="top_ok">
     <property name="geometry">
      <rect>
       <x>1150</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OK!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Comparision of Top10 POIs</string>
    </attribute>
    <widget class="QChartView" name="charts" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>819</width>
       <height>761</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>419</x>
       <y>773</y>
       <width>821</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="laSlider_low">
        <property name="minimum">
         <number>-90</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="value">
         <number>-90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="laSlider_high">
        <property name="minimum">
         <number>-90</number>
        </property>
        <property name="maximum">
         <number>90</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>Current Point's Location:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="valuelabel">
        <property name="text">
         <string>           (NULL)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDateTimeEdit" name="cot_end_edit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>230</width>
       <height>35</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2010</year>
       <month>10</month>
       <day>23</day>
      </datetime>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="cot_start_edit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>230</width>
       <height>35</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2009</year>
       <month>2</month>
       <day>4</day>
      </datetime>
     </property>
    </widget>
    <widget class="QSlider" name="cot_end_time">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>530</y>
       <width>230</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>626</number>
     </property>
     <property name="sliderPosition">
      <number>626</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Start Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>451</y>
       <width>177</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>End Time</string>
     </property>
    </widget>
    <widget class="QSlider" name="cot_start_time">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>230</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>User_2</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cot_user_Box2"/>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>User_1</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cot_user_Box1"/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>388</x>
       <y>12</y>
       <width>52</width>
       <height>761</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSlider" name="cot_lo_high">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="cot_lo_low">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="value">
         <number>-180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Number of Checkins</string>
    </attribute>
    <widget class="QChartView" name="ckinchart" native="true">
     <property name="geometry">
      <rect>
       <x>459</x>
       <y>20</y>
       <width>781</width>
       <height>791</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="ckin_start_time_slider">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>230</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>230</width>
       <height>35</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2009</year>
       <month>2</month>
       <day>4</day>
      </datetime>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Start Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>311</y>
       <width>177</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>End Time</string>
     </property>
    </widget>
    <widget class="QSlider" name="ckin_end_time_slider">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>230</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>626</number>
     </property>
     <property name="sliderPosition">
      <number>626</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Location_ID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="locationBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>230</width>
       <height>35</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2010</year>
       <month>10</month>
       <day>23</day>
      </datetime>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>DAU</string>
    </attribute>
    <widget class="QChartView" name="DAU_1" native="true">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>450</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QChartView" name="DAU_2" native="true">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>450</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>150</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>Location_ID 2</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="dau_lo2"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>360</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>End Time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Start Time</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_4">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>396</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2010</year>
       <month>10</month>
       <day>23</day>
      </datetime>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_3">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>276</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2009</year>
       <month>2</month>
       <day>4</day>
      </datetime>
     </property>
    </widget>
    <widget class="QSlider" name="dau_time_low">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>320</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="dau_time_high">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>440</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>626</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>626</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>80</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>Location_ID 1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="dau_lo1"/>
      </item>
     </layout>
    </widget>
    <widget class="QChartView" name="DAU_3" native="true">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>400</y>
       <width>880</width>
       <height>450</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="step_plus">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>660</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>→(++)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>490</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>TimeStep</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Timestep">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>530</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="step_minus">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>660</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>←(--)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>610</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Step</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stepEdit">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>616</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="warning_text">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>720</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Times New Roman&quot;;
font: 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Sorry, no data contains~</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>900</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialize</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
